### DISCUSSION TITLE
We are using this plug in - is there any way to change format from International to Global Date/Time Stamp

### DISCUSSION THREAD

**Discussion by avenuesrini** [advanced-chat/advanced-chat-components#424] [Q&A]
Right now, Time stamp seems to DD/MM/YY  - which is awesome but of course in America, things are reverse - MM/DD/YY - we want to do "Feb 6, 2:20 PM" - this way its compatible across the board..

Can we change the display to the Mon, Day, YY and time stamp? 

![image](https://user-images.githubusercontent.com/53194512/217908672-b715fa3d-332b-4799-828f-e2038c12a547.png)

**Comment by antoine92190** [comment]
You can set `timestamp` in `rooms > lastMessage` and `messages` props however you want, it doesn't need to be a date object, it can be a string.
So you can format the date on your side and pass `timestamp: 'Feb 6, 2:20 PM'`
